query06.sql

```diff
@@ -1,6 +1,12 @@
 /*
   How many trips were shorter than 10 minutes?
+  ANSWER: 46729 for year 2019 and 74958 for year 2020.
 */
 
 -- Enter your SQL query here
-select ...
+SELECT trip_less_than_10min
+FROM(
+  SELECT COUNT(duration) AS trip_less_than_10min FROM indego_trips_2019_q2 WHERE duration < 10
+  UNION
+  SELECT COUNT(duration) AS trip_less_than_10min FROM indego_trips_2020_q2 WHERE duration < 10
```

Comment:
Note that when using a query like this (where you're `union`-ing single values), you should label your rows. Something like:

```sql
  SELECT '2019' AS year,
         COUNT(duration) AS trip_less_than_10min
  FROM indego_trips_2019_q2 WHERE duration < 10
  
  UNION
  
  SELECT '2020' AS year,
         COUNT(duration) AS trip_less_than_10min 
  FROM indego_trips_2020_q2 WHERE duration < 10
```

The reason is that the `union` won't guarantee the order of the results that it returns. Even though you listed 2019 first and 2020 second, the `union` could return them in the opposite order.